lib/diaspora_federation/validators/relayable_validator.rb

```diff
@@ -0,0 +1,11 @@
+module DiasporaFederation
+  module Validators
+    module RelayableValidator
+      def self.included(model)
+        model.class_eval do
+          rule :parent_guid, :guid
```

Comment:
I only ran test for a case when author and parent author is remote. But I don't see any place in the source code where ```author_signature``` check affects execution flow at all.

Actually, I can write some integration tests for this, if I know what is the idea behind it.